stdlib/InteractiveUtils/src/editless.jl

```diff
@@ -113,15 +114,16 @@ function define_default_editors()
     define_editor(r".*") do cmd, path, line
         `$cmd $path`
     end
+    define_editor([r"\bemacs", "gedit", r"\bgvim"]) do cmd, path, line
```

Comment:
Just out of curiosity, changing `r"\bemacs"` to `r"\bemacs\b"` would also work as a fix, right?

Reply:
That still matches `emacs -nw`?

Reply:
If we changed this to `r"\bemacs\b"`, then graphical emacsclient would no longer be matched, so it would end up being caught by the fallback `r".*"` regex which doesn't jump to line.

Reply:
Ah right, thank you both.